src/main/java/kitchenpos/shared/domain/PurgomalumClient.java

```diff
@@ -1,4 +1,4 @@
-package kitchenpos.infra;
+package kitchenpos.shared.domain;
```

Comment:
```suggestion
package kitchenpos.shared.purgomalum.domain;
```
와 같이 패키지를 구성해보면 더욱 좋을 것 같아요 :)
shared 에 여러개의 도메인이 들어올 수도 있으니 한번 더 나누어 봐도 좋을 것 같네요~